data/free worlds start.txt

```diff
@@ -1399,6 +1399,103 @@ mission "FW Pirates 3.1B"
+		`    An idea forms in your head. The root of the pirate problem was never <planet>'s fleet. You've destroyed many pirate ships since you took to the stars, and they just keep coming. Truly, what keeps the piracy going is <planet>'s culture, a culture built on the backs of its slaves and on the payrolls of its pirate cartel bosses in their private palaces. In this moment of clarity, you see a way to eliminate the pirate threat once and for all, but it will mean breaking Tomek's orders and taking the matter into your own hands`
+		choice
+			`	(Follow Tomek's drunken orders and declare <planet> to be the newest member of the Free Worlds.)`
+			decline
+			`	(Liberate <planet>'s slaves, wipe out the cartel palaces, and destroy the pirates' infrastructure.)`
+				`	Hovering over the pirate spaceport, you demand that the pirates immediately free their slaves, abandon their ships on the ground, and evacuate their spaceport. At first, they seem to comply. Then you get a frantic radio call from a surviving rebel ship. A massive pirate armada just jumped in, and another is launching from hidden hangers all over the planet!  A gravelly voice chuckles over the radio. "If you want our ships, come and get them!"`
+				`	Alarms scream. You look at your tactical display and see a massive salvo of missiles coming from the largest pirate fleet you've ever seen. It looks as if the pirate cartels have been holding their elite ships in reserve, which they're launching now. They've also called in a lot of favors with some outside groups. The local defense fleet you already defeated was just the vanguard for this armada. Half the pirates and marauders in the galactic south must be in the system.`
+				`	It also looks as if they've brought some other friends along. You notice a Republic carrier emitting strange energy signatures and launching fighters that are clearly alien in design.`
+				accept
+	on accept
+		set "fw pirates: own way"
+
+	npc kill
+		personality plunders staying
+		government "Pirate"
+		system Shaula
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Southern Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Core Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
+		fleet "Large Northern Pirates"
```

Comment:
Unless all the fleets decide to be interceptor or carrier fleets that should work out fine.